.github/workflows/ci.yml

```diff
@@ -9,15 +9,15 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: freckle/stack-cache-action@v2
       - uses: freckle/stack-action@v3
       - uses: freckle/weeder-action@v1
```

Comment:
Seems like this could use an update too. The modern pattern is (from memory, so something like):

```yaml
- id: stack
  uses: freckle/stack-action@v4
- uses: freckle/weeder-action@v2
  with:
    compiler: ${{ stack.outputs.ghc-version }}
```

Again, this is from memory, so please check through some other projects to find the actual incantation.